composer.json

```diff
@@ -32,7 +32,7 @@
   "prefer-stable": true,
   "require": {
         "oat-sa/generis": "15.20.4",
-        "oat-sa/tao-core": "dev-release-2022-09 as 50.21.5",
+        "oat-sa/tao-core": "50.21.5.1",
```

Comment:
@bartlomiejmarszal I can't see this version number on tao-core. Can you please confirm?